python/trigScint.py

```diff
@@ -53,6 +53,57 @@ def tagger() :
         return digi
 
 
+class TrigScintQIEDigiProducer(ldmxcfg.Producer) :
+    """Configuration for digitizer for Trigger Scintillators's QIE chip"""
+
+    def __init__(self,name) :
+        super().__init__(name,'ldmx::TrigScintQIEDigiProducer','TrigScint')
+
+        self.mean_noise = 0.02
+        self.number_of_strips = 50
+        self.number_of_arrays = 1
+        self.mev_per_mip = 0.4
+        self.pe_per_mip = 100.
+        self.input_collection="TriggerPadUpSimHits"
+        self.input_pass_name="" #take any pass
+        self.output_collection="trigScintQIEDigisUp"
+        self.input_pulse_shape="Expo" # Name of the input pulse class
+        self.expo_k=0.1          # Inverse of decay time of piece-wise exponential 
+        self.expo_tmax=5.0       # Time at which piece-wise exponential peaks
+        self.maxts=5             # No. of time samples to analyze
+        self.toff_overall = 30.0 # Global time offset
+        self.tdc_thr = 3.4       # Threshold current in uA for TDC latch
+        self.pedestal= 6.0       # QIE pedestal value (in fC)
+        self.elec_noise = 1.5    # Electronic noise (in fC)
+        self.sipm_gain = 1.e6    # SiPM Gain
+        self.qie_sf = 40.        # QIE sampling frequency in MHz
+
+        import time
+        self.randomSeed = int(time.time())
```

Comment:
Is this still used? We're moving to making random number generator seeding deterministic (using the run number) by the use of RNSS (which you have also implemented).